toolkit/tools/pkg/imagecustomizerlib/bootcustomizer.go

```diff
@@ -88,9 +87,11 @@ func (b *BootCustomizer) getSELinuxModeFromGrub() (imagecustomizerapi.SELinuxMod
 
 	// Get the SELinux kernel command-line args.
 	if b.isGrubMkconfig {
-		_, args, _, err = GetDefaultGrubFileLinuxArgs(b.defaultGrubFileContent, defaultGrubFileVarNameCmdlineForSELinux)
+		// Check both GRUB_CMDLINE_LINUX and GRUB_CMDLINE_LINUX_DEFAULT variables
+		// and merge arguments from both if they exist
+		args, err = GetDefaultGrubFileLinuxArgsFromMultipleVars(b.defaultGrubFileContent)
 		if err != nil {
-			return "", err
+			return "", fmt.Errorf("failed to find SELinux args in grub file (%s): %w", installutils.GrubDefFile, err)
```

Comment:
Only adding an extra error message to this `if err != nil` check seems odd. Should there be symmetry with the `else` block? Perhaps add an extra error message in `GetSELinuxMode` against the call to `getLinuxCommandLineArgs`?